### PR TITLE
[Platform support] Update notes

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: docs/notes-base/platforms.md [hunk 1]
```diff
@@ -85,12 +86,13 @@ Android deployment is possible from Windows, Linux and Mac OSX.
 The Android engine supports devices using ARMv7 or ARMv8 processors.
```
Comment (runrevmark) [inline docs/notes-base/platforms.md]: THis is out of date too isn't it?

We now support x86, x86-64, ARM and ARM64

**Review by runrevmark — CHANGES_REQUESTED** [review]

File: docs/notes-base/platforms.md (hunk 1 above)
Comment (livecodepanos) [reply docs/notes-base/platforms.md]: Oops :)

PR updated!

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 3981065

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 3981065bb4659dd8b3ee47b62ccf69a204456362

**Comment by livecode-vulcan** [comment]
😎 test success 3981065bb4659dd8b3ee47b62ccf69a204456362

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/757)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/166)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/166)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/166)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/166)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2283)
* try-community-universal-ios-iphoneos13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.4/builds/17)
* try-community-universal-ios-iphonesimulator13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.4/builds/17)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2312)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2299)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2298)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2333)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2273)

**Pull request merged**